runconfig/parse.go

```diff
@@ -375,6 +382,23 @@ func convertKVStringsToMap(values []string) map[string]string {
 	return result
 }
 
+func parseLoggingOpts(loggingDriver string, loggingOpts []string) (map[string]string, error) {
+	loggingOptsMap := convertKVStringsToMap(loggingOpts)
+	if loggingDriver == "none" && len(loggingOpts) > 0 {
+		return map[string]string{}, fmt.Errorf("Invalid logging opts for driver %s", loggingDriver)
+	}
+	//validation step
```

Comment:
@ibuildthecloud I wanted to enforce a validation step at this point for the log options. The commented lines have a pattern for validating log options.